integration_tests/tests/consistency/consistency_users.sql

```diff
@@ -0,0 +1,47 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+-- this test ensures the iterable__users end model matches the prior version
+with prod as (
+    select *
+    from {{ target.schema }}_iterable_prod.iterable__users
+    where date(created_at) < date({{ dbt.current_timestamp() }})
```

Comment:
When trying to run this test I received this error.

```
10:02:09    Database Error in test consistency_users (tests/consistency/consistency_users.sql)
  Unrecognized name: created_at; Did you mean updated_at? at [12:16]
  compiled Code at target/run/iterable_integration_tests/tests/consistency/consistency_users.sql

```

When looking at `iterable__users`, looks like `created_at` should be changed to `updated_at`.

Reply:
Thanks for catching!